app/src/main/java/fi/bitrite/android/ws/ui/MapFragment.java

```diff
@@ -494,7 +493,7 @@ private void addUserToCluster(SimpleUser user) {
             marker.setAnchor(UserMarker.ANCHOR_CENTER, UserMarker.ANCHOR_BOTTOM);
             marker.setIcon(getResources().getDrawable(R.drawable.map_markers_single));
             marker.setOnMarkerClickListener((m, mapView) -> {
-                new MultiUserSelectDialog().show(Lists.newArrayList(user));
+                new MultiUserSelectDialog().show(new ArrayList<>(Collections.singletonList(user)));
```

Comment:
`Collections.singletonList(user)` instead of `new Arraylist<>(Collections.singletonList(user))` should suffice here